region/maridia/outer/Mama Turtle Room.json

```diff
@@ -886,6 +903,52 @@
       ],
       "flashSuitChecked": true,
       "note": "To save energy, start from the upper right ledge; it may require repositioning Mama Turtle to the right to get to the ledge."
+    },
+    {
+      "link": [5, 6],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
+        "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 3],
+      "name": "Base",
+      "requires": []
+    },
+    {
+      "link": [6, 5],
+      "name": "Base",
+      "requires": [
+        {"and": [
+          "canUseGrapple",
+          {"or": [
+            "canCarefulJump",
+            "HiJump",
+            "Walljump"
+          ]}
+        ]},
```

Comment:
Okay, so in testing this... this base grapple from ledge is slightly tricky as the block is barely offscreen at the moment you have to fire grapple.  It's a lot easier if you have HiJump or Walljump.  This setup puts it in Medium, but we could replace canCarefulJump -> canPreciseGrapple to put it in hard.  It matters mostly because if you can't take the 6->5->1 route to safety, you have to 6->3->1 through Mama Turtle.